src/evaluator.ts

```diff
@@ -1,15 +1,16 @@
 import { execSync } from "node:child_process";
```

Comment:
Will be moving evaluator-related code, including the diff stats stuff, into its own subdir, in another PR